package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "backend",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "type": "module",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "nodemon index.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/pastegram/backend.git"
+  },
+  "keywords": [],
+  "author": "pastegram",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/pastegram/backend/issues"
+  },
+  "homepage": "https://github.com/pastegram/backend#readme",
+  "dependencies": {
+    "cors": "^2.8.5",
+    "express": "^4.17.2",
+    "mongoose": "^6.1.2",
+    "nodemon": "^2.0.15"
```

Comment:
change nodemon into a dev dependency. we don't need that in production
`npm i -D nodemon`